schema/schema.go

```diff
@@ -0,0 +1,73 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/conduitio/conduit-commons/schema"
+	"github.com/conduitio/conduit-connector-protocol/pconduit"
+	"github.com/conduitio/conduit-connector-protocol/pconduit/v1/client"
+	"github.com/conduitio/conduit-connector-sdk/internal"
+	"google.golang.org/grpc"
+)
+
+func init() {
+	internal.StandaloneConnectorUtilities = append(internal.StandaloneConnectorUtilities, standaloneInitializer{})
+}
+
+// Service is the schema service client that can be used to interact with the schema service.
+// It is initialized with an in-memory service by default.
+var Service = newInMemoryService()
+
+// Create creates a new schema with the given name and bytes. The schema type must be Avro.
+func Create(ctx context.Context, typ schema.Type, name string, bytes []byte) (schema.Schema, error) {
+	if typ != schema.TypeAvro {
+		return schema.Schema{}, fmt.Errorf("type %v is not supported (only Avro is supported)", typ)
+	}
+
```

Comment:
Hmm, is this the place to make the call about which schema type is supported? I'd let the schema service make the decision.

So if we want to have this check, we should add it in the in-memory service IMO. And then we can also return `ErrInvalidSchemaType`, since we already have it defined.

Reply:
@lovromazgon good call.